remotefile2.txt

```diff
@@ -1 +1,2 @@
 To test fetch option.
+My very interesting feature addition!!!
```

Comment:
WIll this work. Will have to check. @creativeab88

Reply:
The changes requested are done